releases/dev/app.yml

```diff
@@ -15,6 +15,10 @@ spec:
   rollback:
     enable: true
   values:
+    globals:
+      domainNames:
+        appealsWebApp: appeals-dev.planninginspectorate.gov.uk
```

Comment:
```
➜  appeal-planning-decision git:(feature/AS-1703) ✗ helm lint charts/app
==> Linting charts/app
[ERROR] templates/: template: app/templates/appealsServiceApi.yaml:55:40: executing "app/templates/appealsServiceApi.yaml" at <.Values.globals.domainNames.appealsWebApp>: nil pointer evaluating interface {}.domainNames

Error: 1 chart(s) linted, 1 chart(s) failed
```

So, at a guess, this also needs the config adding:

```
# charts/app/values.yaml

globals:
  domainNames:
    appealsWebApp: http://localhost:9003/
```

That does satisfy helm lint, **but** that doesn't mean it's the correct config. I'm not sure, that's just my best guess.

Reply:
You're almost there guys. I've just refreshed my memory on this stuff.
https://helm.sh/docs/chart_template_guide/values_files/

In order for this to work, you would need another dev specific values file. Something like this:
helm lint charts/app --values charts/app/values.yaml --values releases/dev/app-dev-values.yaml

However, this would need a change to the flux install, to instruct it to read the new environment specific values files. I'll write up a user story. For now, if you want to get this out, you will have to update the default values.yaml, as Chris mentioned. Unfortunately, as it's currently configured, this will update all three environments.

Reply:
👍